Improved 'Amplify CLI configuration without AWS' steps (#2973)
aws-amplify/docs: docs/console/adminui/extend-cli.md

![autologin](~/images/console/cli-pull.png)

> 2. Open a terminal window and navigate to the directory where you would like to have amplify install your project. `amplify pull --appId xxx --envName xxx` will install your project in whichever directory it is executed in.
> 
> 3. Run the following command, replacing the `xxx` values with your unique `appId` and `envName`. 

Reviewer: Possible rewording:

```suggestion
2. Open a terminal window and navigate to the directory where you would like to have Amplify setup your project.
3. To initialize Amplify in the current directory, run the following command. Replace the `xxx` values with your unique `appId` and `envName`. 
```
